modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -363,6 +363,17 @@ neo4j-admin database import full @/path/to/your/<args-filename> mydb
 For more information, see link:https://picocli.info/#AtFiles[Picocli -> AtFiles] official documentation.
 ====
 
+[NOTE]
+.Using both an option with variable arity and a positional parameter
+====
+When using the option `--additional-config`, for example, `--additional-config neo4j.properties --nodes 0-nodes.csv mydatabase`, the `--nodes` option will behave as greedy and the next option, in this case, `mydatabase`,  will be pulled in via the nodes convertor.
```

Comment:
The description comes across as somehow being about `--additional-config`, which it isn't... it's about any multi-value option, like `--nodes` or `--relationships`. Also it's not specific to the import command, even if this is where it was discovered. Otherwise I think the description is good.

Reply:
@tinwelint, What other multi-value options do we have apart from `--nodes` or `--relationships`? I am trying to figure out if we need to add a similar specific note to some of the other neo4j-admin commands.